src/java/picard/analysis/QualityScoreDistribution.java

```diff
@@ -48,13 +48,37 @@
  * @author Tim Fennell
  */
 @CommandLineProgramProperties(
-        usage = "Program to chart " +
-                "quality score distributions in a SAM or BAM file.",
-        usageShort = "Charts quality score distributions for a SAM or BAM file",
+        usage = QualityScoreDistribution.USAGE_SUMMARY + QualityScoreDistribution.USAGE_DETAIL,
+        usageShort = QualityScoreDistribution.USAGE_SUMMARY,
         programGroup = Metrics.class
 )
 public class QualityScoreDistribution extends SinglePassSamProgram {
-
+    static final String USAGE_SUMMARY = "Program to chart quality score distributions in a SAM or BAM file.  ";
+    static final String USAGE_DETAIL = "  This tool is used for determining the overall \"quality\" of a given run. "+
+            "Provides both chart and table outputs, indicating the range of quality scores and the total" +
+            " numbers of bases corresponding to those scores." +
+            "  Options include plotting " +
+            "the distribution of all of the reads, only the aligned reads, or reads that have passed the Illumina " +
+            "Chastity filter thresholds*.  This filtration process removes the least reliable clusters from the image analysis results. " +
+            "<br /> <br />" +
+            "If the original quality score (OQ) tag is present, histogram will plot both the OQ and the " +
+            "recalibrated quality scores.  The default values are the recalibrated scores obtained from the " +
```

Comment:
"The default values are the recalibrated scores" confused me for a second because usually when we say "the default values bla bla bla" in software documentation, it refers to the value of a command line argument. May be worth rephrasing to say e.g. "By default, blablabla is used"